builtin/credential/oidc/oidctesting/provider.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
Note: This file as a whole shouldn't really be here. That's what https://github.com/coreos/go-oidc/issues/150 has been filed for. :)